src/components/editor/metaform-editor-left-drawer.tsx

```diff
@@ -49,7 +48,9 @@ const MetaformEditorLeftDrawer: React.FC<Props> = ({
   const renderFormTab = () => (
     <FormControl fullWidth>
       <Stack spacing={ 2 } padding={ 1 }>
-        <FormLabel>{ strings.draftEditorScreen.editor.form.versionInfo }</FormLabel>
+        <Typography variant="subtitle1">
```

Comment:
Not part of this PR but remove `padding={ 1 }` from line above this (50). Padding on this particular tab is higher than in others.